source/includes/auto-encryption-options.rst

```diff
@@ -0,0 +1,147 @@
+The following table describes the structure of an
+``AutoEncryptionOptions`` object:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Name
+     - Description
+
+   * - **keyVaultClient**
+     - | **Type:** `MongoClient <{+api+}/classes/MongoClient.html>`__
+       |
+       | **Description**
+       | A ``MongoClient`` configured to connect to
+       | the MongoDB instance hosting your {+key-vault-long+}, used to
+       | read and write your {+dek-long+}s.
+       |
+       | Specify ``keyVaultClient`` if your {+dek-long+}s are in a
+       | different cluster than your encrypted data.
+       |
```

Comment:
Suggestion: could be useful to clarify whether leaving it undefined is required when DEKs are in the same cluster.